### ISSUE TITLE
誤差関数の修正

### ISSUE THREAD

**Issue by haru-256** [haru-256/deeplearning-tensorflow-keras#1]
本書では`00_sin_simple_rnn_tensorflow.py`において，誤差関数は以下の数式に従うように実装されていた．

<img src="https://latex.codecogs.com/gif.latex?\begin{align*}&space;E(\boldsymbol{w})&space;&=&space;\frac{1}{N}&space;\sum_n&space;E_n^T&space;(\boldsymbol{w})&space;\\&space;E_n^T&space;(\boldsymbol{w})&space;&=&space;\frac{1}{2}\left(\hat{y}_n^T(\boldsymbol{w})&space;-&space;y_n^T\right)^2&space;\end{align*}">

しかし，最後の時刻からなる誤差関数ではなく全時刻からなる誤差関数を扱いたい

<img src="https://latex.codecogs.com/gif.latex?\begin{align*}&space;E(\boldsymbol{w})&space;&=&space;\frac{1}{N}&space;\sum_n&space;\sum_t&space;E_n^t&space;(\boldsymbol{w})&space;\\&space;E_n^t&space;(\boldsymbol{w})&space;&=&space;\frac{1}{2}\left(\hat{y}_n^t(\boldsymbol{w})&space;-&space;y_n^T\right)^2&space;\end{align*}">

そこで，`tf.unstack()`を用いた方法で誤差関数をたてようと試みたが，`tf.unstack(t, axis=0)`の場合，データ数Nのところでunstackするがデータ数は`placeholder`にしているので，`tf.unstack()`がエラーを起こす．

どうすれば良いのか?

**Comment by haru-256** [comment]
改善案
- データの与えかた(データを入れる`placeholder`の定義)を変更する.
- TensorFlowではなくてChainerやPyTorchを使ってみる．
